python/cudf/cudf/_lib/datetime.pyx

```diff
@@ -61,29 +61,54 @@ def extract_datetime_component(Column col, object field):
 
     return result
 
-
-def ceil_datetime(Column col, object field):
+#cdef _get_rounding_frequency(object freq):
```

Comment:
TODO: This function needs to be uncommented and fixed (this is why the build passed -- uncommenting this function makes it fail).